src/libmeasurement_kit/nettests/facebook_messenger.cpp

```diff
@@ -0,0 +1,26 @@
+// Part of measurement-kit <https://measurement-kit.github.io/>.
+// Measurement-kit is free software. See AUTHORS and LICENSE for more
+// information on the copying conditions.
+
+#include <private/nettests/runnable.hpp>
+
+#include <measurement_kit/nettests.hpp>
+#include <measurement_kit/ooni.hpp>
+
+namespace mk {
+namespace nettests {
+
+FacebookMessengerTest::FacebookMessengerTest() : BaseTest() {
+    runnable.reset(new FacebookMessengerRunnable);
+    runnable->test_name = "facebook_messenger";
+    runnable->test_version = "0.0.1";
+    runnable->needs_input = false;
+}
+
+void FacebookMessengerRunnable::main(std::string input, Settings options,
+                            Callback<Var<report::Entry>> cb) {
```

Comment:
@joelanders the fact that here you receive an `input` that you don't use is a limitation of the implementation of `Runnable`, but in fact the input is ignored, so you should not pass it downstream.

Reply:
Basically here you should have `/*input*/` since we don't need to pass it downstream.

Reply:
done